.githooks/pre-push

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+npm run lint
```

Comment:
We should have a `npm run ci` script which is what the CI will run when we want to validate the new code is merge-able. 

Optionally, we should have this script do that instead.

Reply:
I think in the future, they'll want to be separate. We may want to implement different tests like visual diffing that would be impractical as part of a git hook. The git hook's job is to stop people from wasting time waiting for the CI to run when something simple is gonna fail.

Probably gonna leave this to you as part of a different set of changes (if you intend to be the one setting up the CI pipeline).